app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt

```diff
@@ -308,12 +306,33 @@ class ContactPatientBottomSheet : BaseBottomSheet<
     setAppointmentReminderView.enableNextReminderDateStepper()
   }
 
-  override fun showCallResult(callResult: Outcome, updatedAt: Instant) {
-    // Nothing to look at here, yet.
+  override fun showCallResult() {
+    callPatientView.callResultOutcomeViewVisible = true
   }
 
   override fun hideCallResult() {
-    // Nothing to look at here, yet.
+    callPatientView.callResultOutcomeViewVisible = false
+  }
+
+  override fun setupAgreedToVisitCallResultOutcome() {
+    callPatientView.callResultOutcomeText = getString(R.string.call_result_outcome_agreed_to_visit)
+    callPatientView.setupCallResultViewForAgreedToVisit()
+  }
+
+  override fun setupRemindToCallLaterCallResultOutcome(daysToRemindAppointmentIn: Int) {
+    callPatientView.callResultOutcomeText = resources.getQuantityString(
+        R.plurals.call_result_outcome_remind_to_call_later, daysToRemindAppointmentIn, daysToRemindAppointmentIn.toString()
+    )
+    callPatientView.setupCallResultViewForRemindToCallLater()
+  }
+
+  override fun setupRemovedFromListCallResultOutcome(removeReasonStringRes: Int) {
+    callPatientView.callResultOutcomeText = getString(R.string.call_result_outcome_removed) + getString(removeReasonStringRes)
```

Comment:
What I was suggesting was doing something like this

```kt
val removeReason = getString(removeReasonString)
callPatientView.callResultOutcomeText = getString(R.string.call_result_outcome_removed, removeReason)
```

This way our string can contain a placeholder that indicates that it will be replaced during runtime. WDYT? This is just a nit and can be done in a later PR as well.